pinot-integration-tests/pom.xml

```diff
@@ -406,5 +408,33 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
IMO, the platform-specific tests (spark/hadoop etc) need to be added under the test for each package if we need to add them. 

Also, we should avoid adding all the platform dependencies to the `integration-tests`. Please refer where `SparkSegmentGenerationJobRunnerTest or HadoopSegmentGenerationJobRunnerTest` got added.

We can start with testing the pinot-batch-ingestion-standalone jobs only:
1. SegmentMetadataPushJobRunner
2. SegmentTarPushJobRunner
3. SegmentUriPushJobRunner